src/modules/collectionCuration/actions.ts

```diff
@@ -0,0 +1,73 @@
+export const FETCH_COLLECTION_CURATION_FAILURE = '[Failure] Fetch Collection CollectionCuration'
+
+export const fetchCollectionCurationRequest = (collectionId: string) => action(FETCH_COLLECTION_CURATION_REQUEST, { collectionId })
+export const fetchCollectionCurationSuccess = (collectionId: string, curation?: CollectionCuration) =>
+  action(FETCH_COLLECTION_CURATION_SUCCESS, { collectionId, curation })
+export const fetchCollectionCurationFailure = (error: string) => action(FETCH_COLLECTION_CURATION_FAILURE, { error })
+
+export type FetchCollectionCurationRequestAction = ReturnType<typeof fetchCollectionCurationRequest>
+export type FetchCollectionCurationSuccessAction = ReturnType<typeof fetchCollectionCurationSuccess>
+export type FetchCollectionCurationFailureAction = ReturnType<typeof fetchCollectionCurationFailure>
+
+// Push Collection CollectionCuration
+
+export const PUSH_COLLECTION_CURATION_REQUEST = '[Request] Push Collection CollectionCuration'
+export const PUSH_COLLECTION_CURATION_SUCCESS = '[Success] Push Collection CollectionCuration'
+export const PUSH_COLLECTION_CURATION_FAILURE = '[Failure] Push Collection CollectionCuration'
+
+export const pushCollectionCurationRequest = (collectionId: string) => action(PUSH_COLLECTION_CURATION_REQUEST, { collectionId })
+export const pushCollectionCurationSuccess = () => action(PUSH_COLLECTION_CURATION_SUCCESS)
+export const pushCollectionCurationFailure = (error: string) => action(PUSH_COLLECTION_CURATION_FAILURE, { error })
+
+export type PushCollectionCurationRequestAction = ReturnType<typeof pushCollectionCurationRequest>
+export type PushCollectionCurationSuccessAction = ReturnType<typeof pushCollectionCurationSuccess>
+export type PushCollectionCurationFailureAction = ReturnType<typeof pushCollectionCurationFailure>
+
+// Approve Collection CollectionCuration
+
+export const APPROVE_COLLECTION_CURATION_REQUEST = '[Request] Approve Collection CollectionCuration'
+export const APPROVE_COLLECTION_CURATION_SUCCESS = '[Success] Approve Collection CollectionCuration'
+export const APPROVE_COLLECTION_CURATION_FAILURE = '[Failure] Approve Collection CollectionCuration'
+
+export const approveCollectionCurationRequest = (collectionId: string) => action(APPROVE_COLLECTION_CURATION_REQUEST, { collectionId })
+export const approveCollectionCurationSuccess = (collectionId: string) => action(APPROVE_COLLECTION_CURATION_SUCCESS, { collectionId })
+export const approveCollectionCurationFailure = (collectionId: string, error: string) => action(APPROVE_COLLECTION_CURATION_FAILURE, { collectionId, error })
+
+export type ApproveCollectionCurationRequestAction = ReturnType<typeof approveCollectionCurationRequest>
+export type ApproveCollectionCurationSuccessAction = ReturnType<typeof approveCollectionCurationSuccess>
+export type ApproveCollectionCurationFailureAction = ReturnType<typeof approveCollectionCurationFailure>
+
+// Reject Collection CollectionCuration
```

Comment:
We might want to just say:
```ts
'[XXX] Approve CollectionCuration'
// or
'[XXX] Approve Collection Curation'
```
and
```ts
// Reject Collection Curation
```


here. Same for the other actions. Wdyt?

Reply:
Absolutely, updated!